apps/bfd-data-utilities-fda-drug/pom.xml

```diff
@@ -0,0 +1,174 @@
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>gov.cms.bfd</groupId>
+		<artifactId>bfd-parent</artifactId>
+		<version>1.0.0-SNAPSHOT</version>
+	</parent>
+
+	<groupId>gov.cms.bfd.data.utilities.FDADrug</groupId>
+	<artifactId>bfd-data-utilities-fda-drug</artifactId>
+	<packaging>jar</packaging>
+	<version>1.0-SNAPSHOT</version>
+
+	<description>
+		FDA Drug code lookup jar
+	</description>
+
+	<properties>
+    <maven.javadoc.skip>true</maven.javadoc.skip>
```

Comment:
The issue with your javadoc is that your folder name (FDADrugApp) does not match the package name (FDADrug). If you rename the folder the javadoc plugin will start working and you can remove this (it will then fail for other reasons that should be easy to fix).

I think we should do a bit of further renaming though as well. Java package names should be lowercase and I think we can get away from using the word utility since this package is more about providing the lookup mechanism and just happens to use a utility program as part of its build.

My suggestions is that the maven module be called bfd-data-fda (and later we will have bfd-data-npi) and that the top level package be called gov.cms.bfd.data.fda with two subpackages -- gov.cms.bfd.data.fda.utility (for the utility that builds the drug code file) and gov.cms.bfd.data.fda.lookup (for the actual lookup). We could potentially exclude the utility from the jar that we publish since that is not needed in the server assembly.

Reply:
Done